### PR TITLE
Rn mkhh #29

### PR DESCRIPTION
**Pull request by darkRaf**

### CODE REVIEW

File: src/entities/hr/hr.entity.ts [hunk 1]
```diff
@@ -1,17 +1,21 @@
-import { Entity, PrimaryGeneratedColumn, Column, BaseEntity, PrimaryColumn } from 'typeorm';
+import { Entity, Column, BaseEntity, PrimaryColumn, OneToOne, JoinColumn } from 'typeorm';
 import { hrInterface } from '../types/hr';
+import { User } from '../User/User.entity';
 
 @Entity('hr')
 export class Hr extends BaseEntity implements hrInterface {
   @PrimaryColumn()
   id: string;
+  @OneToOne(() => User, { 'cascade': true })
```
Comment (KonradCencelewicz) [inline src/entities/hr/hr.entity.ts:9]: enter between id and OneToOne. In that case we have Primary colum which isn't generated automaticaly and user column which store user Id. So maybe You should delete id: column and put 'primary': true to OneToOne

File: src/entities/studentsData/studentsData.entity.ts [hunk 2]
```diff
@@ -1,10 +1,14 @@
-import { Entity, Column, BaseEntity, PrimaryColumn } from 'typeorm';
+import { Entity, Column, BaseEntity, PrimaryColumn, OneToOne, JoinColumn } from 'typeorm';
 import { ContractType, StudentStatus, StudentsDataInterface, choiceYesNO, expectedTypeWorkEntity } from '../types/studentsData';
+import { User } from '../User/User.entity';
 
 @Entity('students_data')
 export class StudentsData extends BaseEntity implements StudentsDataInterface {
   @PrimaryColumn()
   id: string;
+  @OneToOne(() => User, { 'cascade': true })
```
Comment (KonradCencelewicz) [inline src/entities/studentsData/studentsData.entity.ts:9]: the same here, enter + primary:true

File: src/entities/hr/hr.entity.ts (hunk 1 above)
Comment (darkRaf) [reply src/entities/hr/hr.entity.ts:9]: Robiłem testy i przy tym zapisie jest ok.

File: src/entities/studentsData/studentsData.entity.ts (hunk 2 above)
Comment (darkRaf) [reply src/entities/studentsData/studentsData.entity.ts:9]: Robiłem testy i przy tym zapisie jest ok.

**Review by KonradCencelewicz — APPROVED** [review]

**Review by JarekKrolik — APPROVED** [review]
OK

**Pull request merged**